delfin/drivers/fujitsu/eternus/eternus_stor.py

```diff
@@ -352,3 +391,548 @@ def parse_alert(context, alert):
     @staticmethod
     def get_access_url():
         return 'https://{ip}'
+
+    def list_storage_host_initiators(self, ctx):
+        initiator_list = []
+        host_status = self.get_host_status()
+        host_fc_list = self.get_data(consts.GET_HOST_WWN_NAMES,
+                                     consts.HOST_TOTAL)
+        for host_fc in (host_fc_list or []):
+            if len(host_fc) < consts.HOST_TOTAL:
```

Comment:
The consts.HOST_TOTAL has been transferred in the self.get_data method. Why does it need to be checked?
![image](https://user-images.githubusercontent.com/59107077/153787768-75b1cb92-3db4-4323-a581-2eba4645bc19.png)

Reply:
The first check is performed to check the number of source data on the device.The second judgment is the verification of the encapsulation result, which is the distrust verification of the encapsulation result.The whole idea is the separation of business logic

Reply:
Here I have changed the names of two different lengths, but it is best to keep them

Reply:
I deleted the first validation, but the two previous validations were done for different data sources